### PR TITLE
Feature/enigma data from clinvar

### PR DESCRIPTION
**Pull request by izcram**
Here the long awaited PR to replace the semi-automated enigma data ingestion with the extraction of equivalent data from ClinVar.

At this point no large changes are expected, except adding a few tests and some minor cleanups.

@melissacline Shall we also remove the enigma code directory?

### CODE REVIEW

File: pipeline/clinvar/enigma_from_clinvar.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+def _get_clinvar_sets(fin):
+    f = etree.parse(fin)
+    root = f.getroot()
+    return root.xpath('//ClinVarSet')
+
+
+def _is_bic_designation(s):
+    return any(k in s.lower() for k in {'ins', 'del', 'dup'}) or \
+           ('>' in s and not s.startswith('p.')) or \
```
Comment (zfisch) [inline pipeline/clinvar/enigma_from_clinvar.py]: if it starts with 'p.' won't it be true anyways?
```suggestion
           ('>' in s or \
```

File: pipeline/clinvar/enigma_from_clinvar.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+            else:
+                abbrev_aa_change = default_val
+
+            return abbrev_aa_change, hgvs_protein
+
+    return default_val, default_val
+
+
+def _extract_assertion_method(enigma_assertion):
+    method_el = enigma_assertion.find(
```
Comment (zfisch) [inline pipeline/clinvar/enigma_from_clinvar.py]: would `.findall()` be useful here?

File: pipeline/clinvar/enigma_from_clinvar.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    condition_id_type = default_val
+
+    if symbol_lst:
+        el = symbol_lst[0]
+
+        condition_id_symbol = clinvar.textIfPresent(el,
+                                                    'ElementValue')  # e.g. BROVCA2
+        condition_id_id = default_val
+
+        xref_el = el.find('XRef')
```
Comment (zfisch) [inline pipeline/clinvar/enigma_from_clinvar.py]: would `.findall()` be useful here?

File: pipeline/clinvar/enigma_from_clinvar.py [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+    rec["Abbrev_AA_change"], rec["HGVS_protein"] = _compute_protein_changes(
+        hgvs_cdna_complete, hgvs_util)
+
+    rec["URL"] = clinvar.textIfPresent(enigma_assertion,
+                                       'ClinicalSignificance/Citation/URL')
+
+    rec["Condition_ID_type"], rec[
+        "Condition_ID_value"] = _extract_condition_info(cvs_el)
+
+    trait_set_el = cvs_el.find('ReferenceClinVarAssertion/TraitSet')
```
Comment (zfisch) [inline pipeline/clinvar/enigma_from_clinvar.py]: would`.findall()` be useful here?

Comment (izcram) [reply pipeline/clinvar/enigma_from_clinvar.py]: no, TraitSet appears precisely onceshttps://github.com/ncbi/clinvar/blob/master/xsd/clinvar_public.xsd#L300

File: pipeline/clinvar/enigma_from_clinvar.py (hunk 1 above)
Comment (izcram) [reply pipeline/clinvar/enigma_from_clinvar.py]: yes, I guess. anyways the filter is incorrect, I've fixed in in a subsequent version

**Comment by zfisch** [comment]
Looking much improved! Unless you have any outstanding concerns, merge whenever you're ready and we can try running the pipeline from start to finish with the new LOVD code and ENIGMA/ClinVar code 👍

**Pull request merged**